packages/design-system/src/components/Button/Button.tsx

```diff
@@ -63,20 +120,24 @@ export class Button extends React.PureComponent {
     return attrs;
   }
 
-  componentType() {
+  componentType(): string {
     let component = this.props.component;
+
     if (component === 'button' && this.props.href) {
       // If `href` is provided and a custom component is not, we render `<a>` instead
       component = 'a';
     }
     return component;
   }
 
-  classNames() {
+  classNames(): any {
```

Comment:
[fix] consider using a more specific type here. Looking at the return statement, this function returns a `string`